lib/pdk/config.rb

```diff
@@ -34,7 +34,14 @@ def user
     end
 
     def self.bolt_analytics_config
-      PDK::Config::YAML.new(file: File.expand_path('~/.puppetlabs/bolt/analytics.yaml'))
+      file = File.expand_path('~/.puppetlabs/bolt/analytics.yaml')
+      PDK::Config::YAML.new(file: file)
```

Comment:
I'm not sure if this matters, but the return types here look different.  The happy path returns a `PDK::Config::YAML` object while the unhappy path returns a `PDK::Config::Namespace` object

Reply:
Perhaps we need a helper there for `PDK::Config::YAML.empty` 

so in `pdk/config/yaml.rb`

``` ruby
class << self
  def empty
    PDK::Config::Namespace.new
  end
end
```

OR

just `PDK::Config::YAML.new` ?

Reply:
`PDK::Config::Namespace` is just `PDK::Config::YAML` without the yaml logic, but yeah I'll change it to `PDK::Config::YAML.new` for consistency.